chain/chain/src/missing_chunks.rs

```diff
@@ -144,6 +145,19 @@ impl<Block: BlockLike> MissingChunksPool<Block> {
         }
     }
 
+    pub fn list_blocks_by_height(&self) -> Vec<BlockStatusView> {
+        let mut rtn = Vec::new();
```

Comment:
is `rtn` short for `return`? If it is, let's use `res` or `result`, that's far common:

```
12:38:43|~/projects/nearcore|m/alt_bn128✓
λ rg -F 'rtn =' | wc -l
1

12:38:51|~/projects/nearcore|m/alt_bn128✓
λ rg -F 'res =' | wc -l
467

12:38:56|~/projects/nearcore|m/alt_bn128✓
λ rg -F 'result =' | wc -l
354
```

Reply:
Done.